src/styles/main.scss

```diff
@@ -1,3 +1,5 @@
-body {
-  margin: 0;
-}
+@import "./utils/variables.scss";
+
+@import "./blocks/body.scss";
+@import "./blocks/normalize.scss";
```

Comment:
Import 'normalize.scss' before other styles to ensure correct styling reset.

Reply:
Thanks for your comments, all fixed.